integration_tests/models/staging/stg_orders.sql

```diff
@@ -8,7 +8,7 @@ with source as (
     #}
     select * from {{ ref('raw_orders') }}
     {% if is_incremental() %}
-        where id > (select max(order_id) from {{ this }})
+        where id > (select max(order_id) from {{ this }}) or id = 2
```

Comment:
Could this incremental block be written more generic instead of `id = 2` ?

Reply:
I'll think about it! In one case we are testing added records, and in another we are testing updates.